modules/extensions-support/src/test/java/org/mule/module/extension/internal/runtime/processor/OperationMessageProcessorTestCase.java

```diff
@@ -148,8 +151,12 @@
     private ConfigurationProvider<Object> configurationProvider;
 
     private OperationMessageProcessor messageProcessor;
+
     private String configurationName = CONFIG_NAME;
     private String target = EMPTY;
+
+    @InjectMocks
```

Comment:
it should auto inject the @Mocks like MuleContext into the MuleMetadataManager's _muleContext_ field when some criteria (like type matching) is met